db/migrate/20180618083035_add_visible_to_zone.rb

```diff
@@ -0,0 +1,5 @@
+class AddVisibleToZone < ActiveRecord::Migration[5.0]
+  def change
+    add_column :zones, :visible, :boolean, :default => true
```

Comment:
Prefer `default_value_for`  at the model level over a database-level default.

Reply:
That being said, you will need a data migration to update all existing zones to visible => true.

Reply:
@Fryguy exactly, so why is preferred more difficult solution? database-level default, at least for boolean, was standard for every project I seen before.

Reply:
I thought about it, what about compromise solution? keep `:default => true` as better alternative to data migration and add `default_value_for` to keep your code standards

Reply:
...ok, I disagree with your approach, but we need to finish this RFE, so as you wish